components-core/src/main/java/com/adyen/checkout/components/util/PaymentMethodTypes.kt

```diff
@@ -39,6 +39,7 @@ object PaymentMethodTypes {
     const val ONLINE_BANKING_PL = "onlineBanking_PL"
     const val ONLINE_BANKING_SK = "onlineBanking_SK"
     const val PAY_BY_BANK = "paybybank"
+    const val ACH = "ach"
```

Comment:
We keep this list alphabetically ordered, same goes for other places, I will put a comment on them as well